config/environments/development.rb

```diff
@@ -44,8 +44,6 @@
     end
   })
 
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
Replacing this with a parsed `HostingEnvironment.application_url` is good. We may as well get Foreman to send emails with the right port in links while we're at it, by changing `hosting_environment.rb` line 6:

```ruby
"http://localhost:3000"
```

to 

```ruby
"http://localhost:#{ENV.fetch('PORT', 3000)}"
```